st2packs-runtime/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.8
+FROM stackstorm/st2:3.0dev
```

Comment:
Do we need to change the `FROM` tag for `runtime`?
As it contains `packs` and `virtualenvs` generated in now "fixed" st2 ubuntu xenial builder which we `cp` afterwards.

Not sure what other implications it'll have. But if `runtime` is based on the same image as `builder`, `runtime` would make no sense anymore as it was initially introduced to save the image size.

Reply:
You are 💯 right. Thanks!